recipe/meta.yaml

```diff
@@ -1,34 +1,41 @@
 {% set name = "coin-or-utils" %}
-{% set version = "2.11.9" %}
+{% set version = "2.11.11" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/coin-or/CoinUtils/archive/releases/{{ version }}.tar.gz
-  sha256: 15d572ace4cd3b7c8ce117081b65a2bd5b5a4ebaba54fadc99c7a244160f88b8
+  sha256: 27da344479f38c82112d738501643dcb229e4ee96a5f87d4f406456bdc1b2cb4
   patches:
     - 0001-Patch-for-downstream.patch
 
 build:
   number: 0
-  skip: true  # [win]
   run_exports:
     - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
 
 requirements:
   build:
-    - gnuconfig  # [unix]
+    - gnuconfig
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('fortran') }}
-    - pkg-config
-    - make  # [unix]
+    - {{ compiler('fortran') }}  # [not win]
```

Comment:
I don't understand how this works - how is it possible that we don't need a fortran compiler on windows after all?
The configure script already warns:
```
configure: WARNING: Failed to find a Fortran compiler!
```

Reply:
The project itself has no Fortran, the Fortran compiler is needed when linking to a Blas/Lapack library with Fortran linkage to determine the name-mangling scheme (I'm not really an expert on this). When either not linking to Blas/Lapack at all (it is not technically needed, though performance is better with it) or when linking to a Blas/Lapack with C linkage, no Fortran compiler is needed. Granted, the warning is a little confusing, though it is only a warning, not an error. 

My (now very vague) recollection, pieced together by looking at comments I wrote during earlier attempts, is that we were at first trying to link to Blas/Lapack with Fortran linkage, which first failed because of the lack of a Fortran compiler and then because of the unexpected extension for object files built with flang. 

It's quite possible I failed to see the forest for the trees at that point and got caught up in trying to fix the immediate issue rather than taking a step back to see that this could be avoided altogether by just leaving out the `--with-blas` flag rather than trying to make the Fortran linkage work. I took the presence of those flag for granted for some reason and there's still no way to make the build work with those flags present on WIndows. Even coming back to it this time with fresh eyes, it wasn't until I tried to really isolate why we've able to build on Github Actions and not here that I realized that it was those flags that were the culprit. 

Anyway, sorry for the tail-chasing.